server/index.js

```diff
@@ -33,6 +33,13 @@ app.get('/users/:id', async(req, res) => {
 
     const user = await db.getUser(id);
 
+    if ( user.length === 0 ) {
```

Comment:
without error handling on the db.getUser you'll get an error "cannot get length of undefined", which if it didn't stop execution at the very least is not truthy meaning it wouldn't go into this block and would pass to line 43